src/support.js

```diff
@@ -435,9 +435,24 @@ function loadWebAssemblyModule(binary, flags) {
         if (prop.startsWith('invoke_')) {
           // A missing invoke, i.e., an invoke for a function type
           // present in the dynamic library but not in the main JS,
-          // and the dynamic library cannot provide JS for it. Use
-          // the generic "X" invoke for it.
-          return env[prop] = invoke_X;
+          // and the dynamic library cannot provide JS for it. Generate
+          // a closure to call the underlying dynCall.
+          if (Module.hasOwnProperty(dynCallName)) {
```

Comment:
this doesn't look right, dynCallName is used before the definition on the next line?

Reply:
You're right.  And I suppose that means the test isn't hitting this either...